app.js

```diff
@@ -1,10 +1,16 @@
 const express = require('express')
 const { getTopics } = require('./controllers/topics.controllers')
+const endpoints = require('./endpoints.json')
 
 const app = express()
 
+app.use(express.json())
```

Comment:
We don't need app.use(express.json()) until we test for a post request. Please avoid adding logic which isn't necessary to complete the ticket at hand